test/backend/inference_backend.py

```diff
@@ -380,6 +380,9 @@ def get_test_models():
         "test_logsoftmax_negative_axis_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{-1:{-1}}, CONSTANT_INPUT:{-1}},
         "test_logsoftmax_large_number_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{-1:{-1}}, CONSTANT_INPUT:{-1}},
 
+        # LoopOp
+        "test_loop11_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{}, CONSTANT_INPUT:{}},
```

Comment:
Are you sure `DYNAMIC_SHAPE:{}` will invoke dynamic tests? I remember, for `DYNAMIC_SHAPE` and `CONSTANT_INPUT`, we need to use `DYNAMIC_SHAPE:{-1:{-1}}` to trigger.

Reply:
The DYNAMIC_SHAPE is NOT really tested here.  Need some investigation for this test case. I will come back on this later.

Reply:
The issue fixed.

Reply:
Thanks!